rackspace/lb/v1/nodes/requests.go

```diff
@@ -249,3 +249,36 @@ func ListEvents(client *gophercloud.ServiceClient, loadBalancerID int, opts List
 		return NodeEventPage{pagination.SinglePageBase(r)}
 	})
 }
+
+// FindNodeByIPPort locates a load balancer node by IP and port.
+func FindNodeByIPPort(
```

Comment:
Since it's in the `nodes` package, we can drop the "Node" part of the name. Also, for convention, let's go with "Get" instead of "Find".
s/FindNodeByIPPort/GetByIPPort